.circleci/config.yml

```diff
@@ -172,9 +172,6 @@ jobs:
       - checkout
       - install-dependencies:
           cache-id: solc-js
-      - run:
-          name: install-npm
-          command: npm install
```

Comment:
Was this removed in favor of `install-dependencies`?

Reply:
It's superfluous. `install-dependencies` already either installs required packages or restores them from cache.

I'm not sure why I included this originally, and I suspect this is just a leftover from before I refactored dependency install into a command.

The command that would actually make *some* sense here would be `npm update` which would update to latest version where available (the cached ones get updated only when we bump version or update `package.json` due to some other reason) but this makes CI less predictable and defeats the purpose of caching them to some extent.